editor/src/components/canvas/canvas-strategies/strategies/drag-to-insert-metastrategy.tsx

```diff
@@ -230,10 +239,43 @@ function dragToInsertStrategyFactory(
             },
           )
 
-          return strategyApplicationResult([
-            ...insertionCommandsWithFrames.map((c) => c.command),
-            reparentCommand,
-          ])
+          const newPath = EP.appendToPath(targetParent, insertionSubjects[0].uid)
+
+          const optionalWrappingCommand =
+            insertionSubjectWrapper != null
+              ? [
+                  updateFunctionCommand(
+                    'always',
+                    (editorState, lifecycle): Array<EditorStatePatch> =>
+                      foldAndApplyCommandsInner(
+                        editorState,
+                        [],
+                        [
+                          wrapInContainerCommand(
+                            'always',
+                            newPath,
+                            wrapperUID,
+                            insertionSubjectWrapper,
+                          ),
+                        ],
+                        lifecycle,
+                      ).statePatches,
+                  ),
+                ]
+              : []
+
+          return strategyApplicationResult(
+            [
+              ...insertionCommandsWithFrames.map((c) => c.command),
+              reparentCommand,
+              ...optionalWrappingCommand,
+            ],
+            {
+              duplicatedElementNewUids: {
```

Comment:
I'd suggest adding a new field to the `CustomStrategyState` specifically for this

Reply:
implemented in https://github.com/concrete-utopia/utopia/pull/3531/commits/e37e17fcacbcf21d51aea361f40a23d74d203ccc